cmake/os_detection.cmake

```diff
@@ -57,4 +58,4 @@ endmacro(DEFINE_OS)
 
 # We catually call the macro upon include of this cmake file so the OS is always
 # defined. This avoid verbosity in the CMakeLists.txt + potential error.
-DEFINE_OS()
\ No newline at end of file
+DEFINE_OS()
```

Comment:
Seems that you forgot to modify this file, see above as well.

Reply:
oops, thanks for catching this